src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,123 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double GROWTH_FACTOR = 1.5;
+    private Object[] values;
```

Comment:
Use generic type here. 
```suggestion
    private T[] values;
```